Assets/MRTK/SDK/Features/UX/Scripts/Pointers/SpherePointer.cs

```diff
@@ -108,8 +154,13 @@ public bool IgnoreCollidersNotInFOV
 
         private void Awake()
         {
-            queryBufferNearObjectRadius = new SpherePointerQueryInfo(sceneQueryBufferSize, NearObjectRadius);
-            queryBufferInteractionRadius = new SpherePointerQueryInfo(sceneQueryBufferSize, SphereCastRadius);
+            InitQueryParameters();
+        }
+
+        public void InitQueryParameters()
```

Comment:
Making this `public` seems slightly odd to me, especially since it seems like it's only called outside this class from the test.
Could we instead make the properties re-init or update the corresponding `SpherePointerQueryInfo`s when they're set?

Reply:
We could maybe add setters for any value that might need updating on the `SpherePointerQueryInfo`, so we don't have to recreate them and can just update specific values? Especially since that class is `private`, so making any public properties still won't allow others to mess with things. And it seems like the class is pretty deterministic and doesn't depend on previous state.